cmd/kubelet/app/options/options.go

```diff
@@ -101,7 +101,7 @@ type KubeletFlags struct {
 	// experimentalMounterPath is the path of mounter binary. Leave empty to use the default mount path
 	ExperimentalMounterPath string
 	// This flag, if set, will avoid including `EvictionHard` limits while computing Node Allocatable.
-	// Refer to [Node Allocatable](https://git.k8s.io/community/contributors/design-proposals/node/node-allocatable.md) doc for more information.
```

Comment:
link all the node-allocatable fields to https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable instead?